flow-typed/Claim.js

```diff
@@ -191,32 +191,3 @@ declare type ChannelPublishParams = {
   tags?: Array<string>,
```

Comment:
:small_red_triangle: Can't publish changes made to a Public Playlist

- Created and publish a playlist with:
    - name: `no-channel`
    - channel: anonymous (my account had no channels)
    - no thumbnail
    - description: `no-channel-description`
    - Items: 1 video
- After published and confirmed, edit it:
    - title: `My Awesome Playlist`
    - thumbnail: Upload one
    - Click Submit (aside, it should probably be Save rather than Submit, since no API is called)
- Click "Publish Updates" -- now stuck:

<img width="571" alt="image" src="https://user-images.githubusercontent.com/64950861/188559975-3a9de9c2-d6f2-4bc4-bf49-35420c71cbb4.png">

- [ ] The wallet also doesn't seem to include the new title, which Miko originally filed. It did include the thumbnail

<img width="471" alt="image" src="https://user-images.githubusercontent.com/64950861/188560334-10f00b4b-4ff9-48bb-b13c-eee3d599f102.png">